website/src/styles/layout/content.scss

```diff
@@ -3,8 +3,11 @@
 $transition-timing: 300ms;
 
```

Comment:
layout on the component detail page is a little bit broken on a smaller resolution (< 600px)

<img width="509" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/362041c5-6dec-4ebe-a23b-a0e3c0ccf521">